### ISSUE TITLE
Fix build errors, broken links, accessibility violations for docs and guides for May, 2020.

### ISSUE THREAD

**Issue by chirp1** [OpenLiberty/docs#1501]
- Fix build errors. They can be documentation that you created or somebody else created. 
- Fix broken links in any topics and any guides. 
- Fix accessibility violations.

**Comment by dmuelle** [comment]
Checked links and ran the DAP accessibility tool on all published pages in docs/ref/general. Fixed all broken links and accessibility violations in #1516 for `draft` and #1519 for `develop`, which should go to `master` with the 20.0.0.5 release on 5/7. There are several potential accessibility violations that occur on every page, related to the framework:


- Combine color and descriptive markup to convey information.
- Provide a mechanism to bypass blocks of content that are repeated on multiple Web pages.?
- Do not use ::before and ::after pseudo-elements to insert non-decorative content.

These potential violations seem to be triggered by elements of the stylesheet, not docs content. We should run the tool again after the transition to Antora to see if these issues persist or if any new ones show up.
